server/app/views/admin/migration/AdminImportViewPartial.java

```diff
@@ -32,7 +40,36 @@ public DomContent renderError(String errorMessage) {
   }
 
   /** Renders the correctly parsed program data. */
-  public DomContent renderProgramData(String programData) {
-    return div(programData).withId(PROGRAM_DATA_ID);
+  public DomContent renderProgramData(ProgramMigration programMigration) {
+    ProgramDefinition program = programMigration.getProgram();
+    DivTag programDiv = div().withId(PROGRAM_DATA_ID);
+    programDiv.with(h3("Program name: " + program.localizedName().getDefault()));
+    programDiv.with(p("Admin name: " + program.adminName()));
```

Comment:
optional nit: this is slightly cleaner (and similar reformatting is possible in `renderProgramBlock()` and `renderQuestion()`:

```suggestion
    DivTag programDiv = div().withId(PROGRAM_DATA_ID)
        .with(
            h3("Program name: " + program.localizedName().getDefault())
            p("Admin name: " + program.adminName()));
```